python/ml4ir/base/model/relevance_model.py

```diff
@@ -348,17 +348,16 @@ def save(
         model_file = os.path.join(models_dir, "final")
 
         # Save model with default signature
-        saved_model.save(self.model, export_dir=os.path.join(model_file, "default"))
+        self.model.save(filepath=os.path.join(model_file, "default"))
 
         """
         Save model with custom signatures
 
         Currently supported
         - signature to read TFRecord SequenceExample inputs
         """
-        saved_model.save(
-            self.model,
-            export_dir=os.path.join(model_file, "tfrecord"),
+        self.model.save(
```

Comment:
Unrelated syntax change. Should be hitting the same API. I read that some memory improvements can be expected. But I didn't observe any. Still clean to keep it this way because both save and load are not keras model APIs - so there's no mismatch.